### DISCUSSION TITLE
Replace prejudiced terms with neutral ones

### DISCUSSION THREAD

**Discussion by paulochf** [online-ml/river#761] [Ideas]
For many years, computer science (like other segments of the society) used racist terms like black/white list and master/slave. Still, recently, some initiatives have been replacing them with others that don't reference racism or slavery.  One example is [Github, which deprecated the `master` branch in favor of `main`].

Tonight I was reading some links I collected for my thesis project. This repo caught my attention because it seems pretty valuable to what I plan to develop. While reading the docs, I stumbled upon [this parameter in Discard transformer], which is the only example I've found in this project that you could improve regarding the topic. Replacing it with `exclusion_list` should be enough, considering any means the maintainers here would make this graceful transition legacy-wise.

Thanks for reading this suggestion. Congrats on the excellent work, and I'm looking forward to seeing this happening to cite the package in my work.

[Github, which deprecated the `master` branch in favor of `main`]: https://www.zdnet.com/article/github-to-replace-master-with-alternative-term-to-avoid-slavery-references/
[this parameter in Discard transformer]: https://riverml.xyz/latest/api/compose/Discard/#parameters

**Comment by MaxHalford** [comment]
Hello 👋 

Sure, I don't mind changing these terms if you find them offensive. I think we can find a better term than `exclusion_list` though.

Do you want to make a pull request, or do you want me to do it?

**Reply by paulochf** [reply]
Hello!

I think it's better you do it. I'm not aware of how you would change such name keeping backward compatibility or even how do you plan to release it. I guess it will require a major version, won't it?

**Comment by MaxHalford** [comment]
Papertrail: addressed in #766

**Reply by paulochf** [reply]
I haven't suffered any prejudice myself, but I know people who did. It's life-changing happening in their lives for the worst.

But I sympathize with them, and this kind of action greatly helps ease these feelings, so thank you so much for hearing the suggestion and making a better world!
